util/expand_params.go

```diff
@@ -65,7 +65,7 @@ func expandMap(inputMap reflect.Value, expansions *Expansions) error {
 		case reflect.String:
 			expandedValString, err := expansions.ExpandString(val.String())
 			if err != nil {
-				return errors.Wrapf(err, "expanding value '%v'", val.String())
+				return errors.Wrapf(err, "expanding value '%s'", val.String())
```

Comment:
Did we test this? I would have thought the string formatter wouldn't matter here because `val.String()` returns the string value of the expansion, and formatting it with verbose vs. string formatting wouldn't change that. It seems like it would make more sense/be less dodgy to return the error with the map's key name instead of the value to be expanded (i.e. `"expanding value for key '%s'", key.String()`)

Reply:
I just changed this to be consistent with the lines below when I was skimming the code. But returning the key name in the error messages does make more sense. Updated.